src/models/AppealPermissionKeys.ts

```diff
@@ -0,0 +1,4 @@
+export enum AppealsPermissionKeys {
+    download = '/admin/appeals-download',
```

Comment:
Are there any other keys we should be aware of?

Reply:
As far as I know these are the only keys that we need to add for now

Reply:
These permissions have to be created in mongo. i.e. create role with name `appeals-team` or similar and add the those permissions.